fre/make/createDocker.py

```diff
@@ -72,10 +72,13 @@ def dockerfile_create(yamlfile,platform,target,execute):
                 click.echo("\ntmpDir created in " + currDir + "/tmp")
                 click.echo("Dockerfile created in " + currDir +"\n")
 
-            if run:
-                dockerBuild.build(containerBuild, containerRun)
-            else:
-                sys.exit()
+                # create build script for container
+                dockerBuild.createBuildScript(containerBuild, containerRun)
```

Comment:
Hmm ok, I thought this looked ok but for some reason I'm seeing this when testing:
```
AttributeError: 'container' object has no attribute 'createBuildScript'
```

Reply:
WAIT, I think I forgot to install the recent changes in my conda environment